README.rst

```diff
@@ -6,17 +6,10 @@
 Cirq is a Python library for writing, manipulating, and optimizing quantum
 circuits and running them against quantum computers and simulators.
 
-.. image:: https://travis-ci.com/quantumlib/Cirq.svg?token=7FwHBHqoxBzvgH51kThw&branch=master
-  :target: https://travis-ci.com/quantumlib/Cirq
+.. image:: https://travis-ci.com/chlorophyll-zz/Cirq.svg?branch=master
```

Comment:
i modified the travis.yml because the pylint and mypy tests were failing and therefore the builds were not tested. so i created a travis-ci instance to include the build tests anyway. 

i made the commit after the pull request. 
i did not kniow that this commit was also included in the pull request.
i guess i should have created another branch for that commit. 
i only wanted to include 
missing pylatex package 8063503a79fed3cf3eb4d491940d6bf842692685
and
mypy format errors 252b5fb7dc2e0ccf4bae86fb4bc06c50c1b21ae2
in the pull request. i am not that familiar with branches. i will learn github better and take tutorials.